cli/tests/pcluster/validators/test_iam_validators.py

```diff
@@ -113,3 +115,125 @@ def test_additional_iam_policy_validator(mocker, policy_arn, expected_get_policy
     mocker.patch("pcluster.aws.iam.IamClient.get_policy", side_effect=expected_get_policy_side_effect)
     actual_failures = AdditionalIamPolicyValidator().execute(policy=policy_arn)
     assert_failure_messages(actual_failures, expected_message)
+
+
+@pytest.mark.parametrize(
+    "resource_prefix, expected_message,expected_failure_level",
+    [
+        (
+            r"\pathprefix\\",
+            "Unsupported format for ResourcePrefix",
```

Comment:
If possible, it's always better to test the entire error message that is returned, not just a part of it.

Reply:
This case was giving an exception while comparing due to the the escape character given and which is part of the expected message.

Reply:
Will add a comment for TO DO